src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Enduser/EnduserValidationRules.cs

```diff
@@ -0,0 +1,530 @@
+        /// <param name="useruuid">The UUID of the user performing the action (current user).</param>
+        /// <param name="party">The UUID of the acting party (the user acting on behalf). Can also be "me".</param>
+        /// <param name="from">The UUID of the 'from' party (could be a person or an organization).</param>
+        /// <param name="to">The UUID of the 'to' party (could be a person or an organization).</param>
+        /// <returns>
+        /// A validation rule that adds an error if any UUID is invalid or empty, 
+        /// or if the party does not match either the 'from' or 'to' UUID.
+        /// </returns>
+        public static FuncExpression EnduserReadInputCombination(string party, string from, string to) => () =>
+        {
+            if (!Guid.TryParse(party, out var partyUuid))
+            {
+                return (ref ValidationErrorBuilder errors) =>
+                    errors.Add(ValidationErrors.InvalidQueryParameter, $"QUERY/party", [new("party", $"The 'party' parameter is not valid.")]);
+            }
+
+            if (!string.IsNullOrEmpty(from) && Guid.TryParse(from, out var fromUuid) && fromUuid == partyUuid)
+            {
+                return null;
+            }
+
+            if (!string.IsNullOrEmpty(to) && Guid.TryParse(to, out var toUuid) && toUuid == partyUuid)
+            {
+                return null;
+            }
+
+            return (ref ValidationErrorBuilder errors) =>
+            {
+                errors.Add(ValidationErrors.InvalidQueryParameter, $"QUERY/from", [new("from", "Either the 'from' UUID or the 'to' UUID must match the 'party' UUID. Neither matches the 'party' UUID.")]);
+                errors.Add(ValidationErrors.InvalidQueryParameter, $"QUERY/to", [new("to", "Either the 'to' UUID or the 'from' UUID must match the 'party' UUID. Neither matches the 'party' UUID.")]);
+            };
+        };
+
+        /// <summary>
+        /// Delete combination
+        /// </summary>
+        /// <param name="useruuid"></param>
+        /// <param name="party"></param>
+        /// <param name="from"></param>
+        /// <param name="to"></param>
+        /// <returns></returns>
+        public static FuncExpression EnduserRemoveCombination(string party, string from, string to) => () =>
+        {
+            if (!Guid.TryParse(party, out var partyUuid))
+            {
+                return (ref ValidationErrorBuilder errors) =>
+                    errors.Add(ValidationErrors.InvalidQueryParameter, $"QUERY/party", [new("party", $"The 'party' parameter is not valid.")]);
+            }
+
+            if (!string.IsNullOrEmpty(from) && Guid.TryParse(from, out var fromUuid) && fromUuid == partyUuid)
+            {
+                return null;
+            }
+
+            return (ref ValidationErrorBuilder errors) =>
+            {
+                errors.Add(ValidationErrors.InvalidQueryParameter, $"QUERY/from", [new("from", "must match the 'party' UUID.")]);
```

Comment:
Validering for sletting må kreve at alle 3: party, from og to er spesifisert + at enten party og from er lik, eller at party og to er lik

Reply:
fixed